### PR TITLE
Create 2018-12-01-bilbao.md

### PR DESCRIPTION
**Pull request by usasigain**
We will like to add this new Clojure Bridge event to the official calendar. Let us know if something is missing

### CODE REVIEW

File: _posts/2018-12-01-bilbao.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+permalink:          
+country:            Spain
+city:               Bilbao
+street:             Uribitarte Kalea, 6, Paseo Uribitarte, 3, 48001 Bilbao, Vizcaya
+latitude:           43.264572
+longitude:          -2.926802
+start-date:         2018-12-01
+end-date:           2018-12-01
+registration-url:   https://www.eventbrite.es/e/entradas-clojure-bridge-bilbao-50914127526?utm_campaign=new_event_email&utm_medium=email&utm_source=eb_email&utm_term=viewmyevent_button
+city-image-url:
```
Comment (davidchambers) [inline _posts/2018-12-01-bilbao.md]: Could you choose a [city photo][1]? :)


[1]: https://www.flickr.com/search/?l=comm&q=Bilbao

Comment (usasigain) [reply _posts/2018-12-01-bilbao.md]: Done!

File: _posts/2018-12-01-bilbao.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+#### Who Should Attend?
+
+ClojureBridge is open to women and all trans*, genderqueer, and gender non-conforming people that are interested in learning about computer programming. For software developers that are new to the Clojure language we offer a Track 2 to dive right in to functional programming and concurrency.
+
+#### Details
+
+In Track 1 we'll take you through building a sample application using Clojure. At the end of the workshop your application will be live on the Internet!
+
+In Track 2 we'll cover basic syntax and data structures and discuss functional programming and concurrency concepts.
```
Comment (davidchambers) [inline _posts/2018-12-01-bilbao.md]: Will you provide two tracks, or just one?

Comment (usasigain) [reply _posts/2018-12-01-bilbao.md]: We updated the definition.

File: _posts/2018-12-01-bilbao.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+gravatar-email:     
+organizers:         [{"email": "info@magnet.coop", "github": "magnetcoop", "name": "Magnet", "twitter": "magnetcoop"}]
+sponsors:           [{"image": "", "name": "Magnet", "url": "https://www.magnet.coop/"}]
+---
+
+ClojureBridge aims to increase diversity within the Clojure community by offering free, beginner-friendly Clojure programming workshops for underrepresented groups in tech.
+
+#### Who Should Attend?
+
+ClojureBridge is open to women and all trans*, genderqueer, and gender non-conforming people that are interested in learning about computer programming. For software developers that are new to the Clojure language we offer a Track 2 to dive right in to functional programming and concurrency.
```
Comment (davidchambers) [inline _posts/2018-12-01-bilbao.md]: Is the part about Track 2 applicable to your workshop?

Comment (usasigain) [reply _posts/2018-12-01-bilbao.md]: Sorry David, we missed that part in the previous modification. We think that now it's ok. Thank you!

File: _posts/2018-12-01-bilbao.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+street:             Uribitarte Kalea, 6, Paseo Uribitarte, 3, 48001 Bilbao, Vizcaya
+latitude:           43.264572
+longitude:          -2.926802
+start-date:         2018-12-01
+end-date:           2018-12-01
+registration-url:   https://www.eventbrite.es/e/entradas-clojure-bridge-bilbao-50914127526?utm_campaign=new_event_email&utm_medium=email&utm_source=eb_email&utm_term=viewmyevent_button
+city-image-url:     https://c2.staticflickr.com/2/1900/44197966201_6d024b2ce6_b.jpg
+gravatar-email:     
+organizers:         [{"email": "info@magnet.coop", "github": "magnetcoop", "name": "Magnet", "twitter": "magnetcoop"}]
+sponsors:           [{"image": "", "name": "Magnet", "url": "https://www.magnet.coop/"}]
```
Comment (davidchambers) [inline _posts/2018-12-01-bilbao.md]: Could you provide an image URL for the Magnet logo? If not, we can publish this as is.

Comment (usasigain) [reply _posts/2018-12-01-bilbao.md]: Added! Thank you!

**Comment by davidchambers** [comment]
Thanks, @usasigain. See <https://clojurebridge.org/events/2018-12-01-bilbao>. :)

**Comment by usasigain** [comment]
Thank you very much @davidchambers !!

**Comment by davidchambers** [comment]
You're welcome. I hope you have a successful and enjoyable workshop. :D

**Pull request merged**